src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,437 @@
+analysts: [tomayac]
+editors: [tunetheweb]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1b4moteB9EiLYkH1Ln9qfi1tnU-E4N2UQ87uayWytDKw/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+christianliebel_bio: Christian Liebel is a consultant at <a hreflang="en" href="https://thinktecture.com">Thinktecture</a>, supporting clients from various business areas in implementing great web applications. He is a Microsoft MVP for Developer Technologies, Google GDE for Web/Capabilities and Angular, and participates in the W3C Web Applications Working Group.
+featured_quote: Capabilities are new web platform APIs that unlock entirely new use cases for web applications.
+featured_stat_1: 9%
+featured_stat_label_1: Desktop web sites use the Web Share API
+featured_stat_2: 8.25%
+featured_stat_label_2: Mobile sites use the Async Clipboard API
+featured_stat_3: 11
+featured_stat_label_3: Mobile sites use Declarative Link Capturing
 ---
 
-## TODO
+## Introduction
+Capabilities are new web platform APIs that unlock entirely new use cases for web applications. Those new APIs are essential for [Progressive Web Apps (PWA)](../pwa), a web-based application model. A PWA is a web app that users can install to their system. PWAs run even offline and launch quickly. To integrate with the underlying operating system, PWAs can only use web platform APIs. While browsers have already exposed some lower-level features to the web (e.g., [geolocation](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API), [gamepad](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API), or [webcam](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) access), many APIs were still missing or were clumsy to use (e.g., file system or clipboard access).
+
+### Project Fugu
+
+The <a hreflang="en" href="https://www.chromium.org/teams/web-capabilities-fugu">Capabilities Project</a> (codename Fugu) is a joint effort by Microsoft, Intel, Google, and other Chromium contributors. It tries to bridge the gap between platform-specific applications and web apps by designing and implementing new powerful web platform APIs in a secure and privacy-preserving manner (see also the [Privacy](../privacy) chapter). As capabilities unlock more and more use cases, they lay the path for entire new application categories to finally make the shift to the web (e.g., IDEs, image editors, or office applications).
+
+<figure>
+<blockquote>Project Fugu 🐡 is an effort to close gaps in the web's capabilities enabling new classes of applications to run on the web. […] APIs that Project Fugu is delivering enable new experiences on the web while preserving the web's core benefits of security, low-friction, and cross-platform delivery. All Project Fugu API proposals are made in the open and on the standards track.</blockquote>
```

Comment:
Never seen that `[...]` usage before?

```suggestion
<blockquote>Project Fugu 🐡 is an effort to close gaps in the web's capabilities enabling new classes of applications to run on the web… APIs that Project Fugu is delivering enable new experiences on the web while preserving the web's core benefits of security, low-friction, and cross-platform delivery. All Project Fugu API proposals are made in the open and on the standards track.</blockquote>
```

Reply:
The brackets are used to show that the original text was elided, as opposed to the original text literally containing the ellipsis.

Reply:
Yeah I've always used a literal elipse for that under the assumption that always indicated parts of quotes.

Though I have used brackets when actually doing an edit to a quote:

> Rick said "The brackets are used to show that the original text was [edited], as opposed to the original text literally containing the ellipsis."

But anyway, happy to leave as is, if it's a known convention.

Reply:
https://www.thepunctuationguide.com/ellipses.html#single-sentence

Reply:
Seems like it's a controversial topic in editor circles:
https://english.stackexchange.com/questions/5972/how-are-bracket-ellipsis-used-in-quotations

The things you discover working on this project!

The Google Style Guide (that we use) doesn't seem to mandate square brackets: https://developers.google.com/style/ellipses?hl=en#ellipses-in-text

So I'd vote to remove them.